zilliqa/src/contracts/deposit.sol

```diff
@@ -68,4 +72,12 @@ contract Deposit {
     function getStakers() public view returns (bytes[] memory) {
         return _stakerKeys;
     }
+
+    function getPeerId(bytes calldata blsPubKey) public view returns (bytes memory) {
+        require(blsPubKey.length == 48);
+        if (_stakersMap[blsPubKey].rewardAddress == address(0)) {
```

Comment:
Small nit, but I'd prefer if we checked `_stakersMap[blsPubKey].peerId` here for zero-ness.
We plan on allowing zero addresses later, which will involve removing the check in `getRewardAddress`, but that would cause a problem here (see #803)